core/src/main/java/com/scalar/db/storage/cassandra/ValueBinder.java

```diff
@@ -20,7 +20,7 @@
  * @author Hiroyuki Yamada
  */
 @NotThreadSafe
-public final class ValueBinder implements ValueVisitor {
+public final class ValueBinder implements ColumnVisitor {
```

Comment:
Just a question or topic for discussion. Should the class name be changed to `ColumnBinder`? Or, does it introduce too many changes?

Reply:
I kept this class name `ValueVisitor` because I thought it was still a proper name since this class binds *value* to the statement. What do you think?

> Or, does it introduce too many changes?

I don't think it introduces too many changes.

Reply:
Right. Now, I agree!